Realm/RLMConstants.h

```diff
@@ -94,47 +125,49 @@ typedef NS_ENUM(NSInteger, RLMError) {
 #pragma mark - Notification Constants
 
 /**
- Posted by RLMRealm when the data in the realm has changed.
+ This notification is posted by a Realm when the data in that Realm has changed.
 
- DidChange are posted after a realm has been refreshed to reflect a write
- transaction, i.e. when an autorefresh occurs, `[RLMRealm refresh]` is
- called, after an implicit refresh from `[RLMRealm beginWriteTransaction]`,
- and after a local write transaction is committed.
+ More specifically, this notification is posted after a Realm has been refreshed to
+ reflect a write transaction. This can happen when an autorefresh occurs, when
+ `[RLMRealm refresh]` is called, after an implicit refresh from `[RLMRealm beginWriteTransaction]`,
```

Comment:
This should be `-[RLMRealm refresh]` and `-[RLMRealm beginWriteTransaction]` respectively (leading dash to donate instance method).